spec/queries/content_items_query_spec.rb

```diff
@@ -76,4 +76,22 @@
       expect(results).to match_array(content_items)
     end
   end
+
+  describe "paging results" do
+    it "limits the number of items returned to the maximum per page" do
+      create_list(:content_item, 26)
```

Comment:
Creating a list of 26 items is a bit too much in terms of performance. I would prefer to modify the `Kaminary` page side in a `before` block, and then set the default in an `after` block.
There should some tests with a similar approach in our codebase.